crt_portal/cts_forms/views.py

```diff
@@ -18,11 +18,18 @@ def IndexView(request):
     # formatting protected class
     for report in latest_reports:
         reports = []
-        for p_class in report.protected_class.all().order_by('-form_order'):
+        for p_class in report.protected_class.all().order_by('form_order'):
             if p_class.protected_class is not None:
-                reports.append(PROTECTED_CLASS_CODES.get(p_class.protected_class, p_class.protected_class))
+                code = PROTECTED_CLASS_CODES.get(p_class.protected_class, p_class.protected_class)
+                if code is not None:
+                    reports.append(code)
+                elif report.other_class is None:
```

Comment:
I don't understand why we check for `report.other_class` being `None` only if `code is not None` — the two seem logically independent to me.

Reply:
We refactored this and added a code comment to explain!